infection.json.dist

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "vendor/infection/infection/resources/schema.json",
+    "source": {
+        "directories": [
+            "lib"
+        ]
+    },
+    "mutators": {
+        "@default": true,
+        "IncrementInteger": {
+            "ignoreSourceCodeByRegex": [
+                ".*json_decode.*"
```

Comment:
`512` is specified as we need to specify `\JSON_THROW_ON_ERROR`, Infection wants to mutate this but it's difficult to test the impact this has due to the data we're serializing.